src/elexmodel/handlers/data/VersionedData.py

```diff
@@ -64,10 +65,12 @@ def get_versioned_results(self, filepath=None):
 
         if self.election_id.startswith("2020-11-03_USA_G"):
             path = "elex-models-prod/2020-general/results/pres/current.csv"
-        elif self.election_id.startswith("2024-11-05_USA_G"):
```

Comment:
why has this been removed?

Reply:
It wasn't, the logic was a little redundant here so I cleaned it up right below this 😄